integration_test/third_party_apps_data/applications/mssql/metadata.yaml

```diff
@@ -38,11 +38,13 @@ expected_metrics:
   monitored_resource: gce_instance
   labels: {}
 # v2 metrics
-- type: workload.googleapis.com/sqlserver.user.connection.count # TODO: look into this metric type change
-  value_type: DOUBLE
+- type: workload.googleapis.com/sqlserver.user.connection.count
+  value_type: INT64
   kind: GAUGE
   monitored_resource: gce_instance
   labels: {}
+  # if no user connection may not be present
```

Comment:
It may be best to make this an end-of-line comment, to avoid it being accidentally separated from the value.

Reply:
Actually, can/should we force a user connection to happen as part of the test? This may be more of a follow-up thing, though…

Reply:
From my testing the current integration test does create a single user connection on the instance, I just noticed this while doing some deeper testing on my own sqlserver instance that I spun up!

Reply:
Actually I am going to remove the optional, just to prevent future confusion